cpp/src/transform/row_conversion.cu

```diff
@@ -32,15 +31,19 @@
 #include <cudf/utilities/default_stream.hpp>
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/traits.hpp>
+
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/device_buffer.hpp>
 #include <rmm/device_uvector.hpp>
 #include <rmm/exec_policy.hpp>
+
+#include <cooperative_groups.h>
 #include <thrust/binary_search.h>
 #include <thrust/iterator/counting_iterator.h>
 #include <thrust/iterator/discard_iterator.h>
 #include <thrust/iterator/transform_iterator.h>
 #include <thrust/scan.h>
+
 #include <type_traits>
 
 #if defined(__CUDA_ARCH__) && __CUDA_ARCH__ >= 700
```

Comment:
We require Volta+ now, so we can probably remove this part of the macro?
```suggestion
#if defined(__CUDA_ARCH__)
```

Reply:
Let us please not do this here. This change is really substantively only a change of include ordering, so if we wanted to make future "git-annotate" usage easier we could add the single commit to the git blame ignore file so that annotations skip over things. This change, which is substantive, is easily lost in such a scenario.

Reply:
Totally agree - just didn’t want to ignore this. A follow-up PR or opening an issue is appropriate.

Reply:
Please open an issue.